.github/workflows/checks.yml

```diff
@@ -20,3 +20,21 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: Run ESLint
         run: yarn lint
+  test:
```

Comment:
I think we could just add a single step to the job we already have? We would save quite some time because we don't have to run `yarn install` 2 times